### DISCUSSION TITLE
Upgrade all packages with security updates

### DISCUSSION THREAD

**Discussion by berezovskyi** [microsoft/winget-cli#4806] [Ideas]
It would be good to have an extra switch like `--only-security` so that

    winget upgrade --all --only-security

would get upgrades specifically marked as security upgrades (e.g. addressing a CVE). More precisely, this would upgrade all apps that have at least one newer version marked as addressing a security issue. This does not yet seem to be tracked in the manifests.

**Comment by denelon** [comment]
@berezovskyi, we've been looking at:
* https://github.com/microsoft/winget-cli/issues/2204
